packages/js-client/src/internal/modules/methods.ts

```diff
@@ -1,25 +1,42 @@
 import * as BUILD_METADATA from '../../../../contracts/src/build-metadata.json';
+import { MyPlugin__factory } from '../../../../contracts/typechain';
+import { MyPluginContext } from '../../context';
 import {
   NumberListItem,
   NumbersQueryParams,
   NumbersSortBy,
   PrepareInstallationParams,
 } from '../../types';
-import { MyPluginClientCore } from '../core';
 import { QueryNumber, QueryNumbers } from '../graphql-queries';
 import { IMyPluginClientMethods } from '../interfaces';
-import { SubgraphNumber, SubgraphNumberListItem } from '../types';
+import {
+  StoreNumberStep,
+  StoreNumberStepValue,
+  SubgraphNumber,
+  SubgraphNumberListItem,
+} from '../types';
 import { toNumber, toNumberListItem } from '../utils';
 import {
+  ClientCore,
   prepareGenericInstallation,
   PrepareInstallationStepValue,
   SortDirection,
 } from '@aragon/sdk-client-common';
 
 export class MyPluginClientMethods
-  extends MyPluginClientCore
+  extends ClientCore
   implements IMyPluginClientMethods
 {
+  private myPluginRepoAddress: string;
+  private myPluginAddress: string;
+
+  constructor(context: MyPluginContext) {
```

Comment:
I'm not really sure about this one, I understand the deletion of the core for simplyfiying the client but this addresses I think that should be accessible from all modules, not just methods, and replicating this constructor on all methods does not seem to be the optimal thing